src/ExpressionEvaluator/Core/Source/ResultProvider/Helpers/TypeHelpers.cs

```diff
@@ -510,7 +510,14 @@ internal static bool TryGetEvalAttribute<T>(this DkmClrType type, out DkmClrType
                 {
                     result = new Dictionary<string, DkmClrDebuggerBrowsableAttributeState>();
                 }
-                result.Add(browsableAttribute.TargetMember, browsableAttribute.State);
+
+                // There can be multiple same attributes for nested classes.
```

Comment:
>There can be multiple same attributes for nested classes. [](start = 19, length = 57)

Does "nested" mean "derived" here? If so, let's use "derived" in the several places in the comment.